orchagent/fdborch.cpp

```diff
@@ -300,6 +397,47 @@ void FdbOrch::update(sai_fdb_event_t        type,
             }
         }
 
+        // If MAC is MCLAG remote do not delete for age event, Add the MAC back..
+        if (existing_entry->second.origin == FDB_ORIGIN_MCLAG_ADVERTIZED)
+        {
+            sai_status_t status;
+            sai_fdb_entry_t fdb_entry;
+
+            fdb_entry.switch_id = gSwitchId;
+            memcpy(fdb_entry.mac_address, entry->mac_address, sizeof(sai_mac_t));
+            fdb_entry.bv_id = entry->bv_id;
+
+            sai_attribute_t attr;
+            vector<sai_attribute_t> attrs;
+
+            attr.id = SAI_FDB_ENTRY_ATTR_TYPE;
+            attr.value.s32 = SAI_FDB_ENTRY_TYPE_STATIC;
```

Comment:
Why do we set `SAI_FDB_ENTRY_TYPE_DYNAMIC` on `SAI_FDB_EVENT_LEARNED` but `SAI_FDB_ENTRY_TYPE_STATIC` on `SAI_FDB_EVENT_AGED`?

Reply:
me too, why?